### ISSUE TITLE
`related` endpoint hasn't been working for a while now

### ISSUE THREAD

**Issue by FredericEspiau** [uidotdev/npm-trends#190]
Hey,

Thanks for all the hard work, `npm-trends` is one of my favorite tools!

Unfortunately, the `related` endpoint to get the packages related to the current search hasn't been working properly for a while now (a few months).

Sometimes it gets no results, other times it does provide a result but after a very long time.

For example, I was looking for `vitest` and got no related packages.

Cheers and thanks again.

**Comment by R-Mohammed-Hasan** [comment]
yes this was the main reason I was using npmtrends. But it's not available now...

**Comment by ahmedwakil66** [comment]
It was a great tool, why haven’t developers been fixing this? Did they abandon it, or is it that the related data can no longer be collected as it could before?
